### ISSUE TITLE
Publish to npm

### ISSUE THREAD

**Issue by salehe** [dvlsg/async-csp#23]
Hi,

Please make a new release and publish it to npm.

I think at least commit "refactor: make natively runnable in nodejs 7.6+" is worth a new release.

Thanks

**Comment by dvlsg** [comment]
I agree - I want to be careful about breaking backwards compatibility, though. I have fewer concerns, especially since the current LTS version of node (8.9) has async built in, but I'd still rather avoid accidentally breaking someones build.

My original plan was to release the "you need to have native async" version when bumping to 1.0.0, which would assist with indicating the breaking change. I would also like to review some of the other requests for built-in map/filter/reduce/etc capabilities at the same time as 1.0. Been playing with a few potential options offline. 

----

The published version should still be usable on 7.6+, just full of unnecessary polyfills (which results in some performance loss, especially using something like regenerator to cover the generators). Are you running into any problems using it?
